### PR TITLE
Add remove_if to the Data Driven example

### PR DESCRIPTION
**Pull request by jfalcou**
FInal fix for #984 that put remove_if back.

### CODE REVIEW

File: examples/oop/data_driven_physics.cpp [hunk 1]
```diff
@@ -137,10 +138,13 @@ TTS_CASE("Run a ball simulation")
       times.clear();
     }
 
+    // Remove ball that bounced
+    balls.erase(eve::algo::remove_if(balls, [max_bounce](auto b) { return count(b) < max_bounce; }) );
```
Comment (DenisYaroshevskiy) [inline examples/oop/data_driven_physics.cpp:142]: Should be count >= no?

Comment (jfalcou) [reply examples/oop/data_driven_physics.cpp:142]: WIll fix

**Review by DenisYaroshevskiy — APPROVED** [review]

File: examples/oop/data_driven_physics.cpp [hunk 2]
```diff
@@ -120,10 +121,13 @@ TTS_CASE("Run a ball simulation")
 
     // Display 200 balls
     for(std::size_t i=0;i<200;i++)
```
Comment (DenisYaroshevskiy) [inline examples/oop/data_driven_physics.cpp:123]: just up to the ball size would be simpler but w/e

Comment (jfalcou) [reply examples/oop/data_driven_physics.cpp:120]: if you put 10000 balls , it doesn't fit the screen. I just plot the 200 first

**Pull request merged**